bamboo/bootstrap-unit-tests.sh

```diff
@@ -9,19 +9,18 @@ set -ex
 export SSH_USERS=user:$(id -u):$(id -u)
 export COMPOSE_FILE=./bamboo/docker-compose.yml
 
-## Set container_id for docker-compose to use to identify the compose stack per planKey
-docker_command="docker exec -t ${container_id}_build_env_1 /bin/bash -c"
+## Set container_id for docker compose to use to identify the compose stack per planKey
```

Comment:
Hmm should we be seeing code changes here? I think `release-18.0.x` should be current with `master` and the `release-18.0.0` branch should just be docs and version bump? Was there a commit cherry-picked or something here?

Reply:
Same as previous comment. I don't think we necessarily need to fix anything. The release-18.0.x branch should end up in the state we want when this is merged.